### PR TITLE
fixed button in wallet-overview

### LINKED ISSUE #19921: Use truncation and ellipsis for text overflow for global action labels
The action button's text overflow renders labels to overlap each other when translated to different languages. We should fix this by a combination of truncation and ellipsis.

### PR DESCRIPTION
**Pull request by NidhiKJha**
This PR is to update the portfolio button label for different languages

Fixes #19921 

### Before
Screenshot 2023-07-07 at 6 27 22 PM

### After
Screenshot 2023-07-07 at 6 27 05 PM

## Manual Testing Steps

- Switch to multiple languages
- Go to home screen
- Check the button text doesn't overflow in wallet overview

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by kevinghim** [comment]
Will resolve https://github.com/MetaMask/metamask-extension/issues/19921

**Review by vthomas13 — DISMISSED** [review]

File: ui/components/ui/icon-button/icon-button.scss [hunk 1]
```diff
@@ -29,4 +29,8 @@
     opacity: 0.3;
     cursor: auto;
   }
+
+  p {
+    width: 60px; //for ellipsis keeping the width same as icon-button used here
```
Comment (darkwing) [inline ui/components/ui/icon-button/icon-button.scss:34]: Should this selector be `.icon-button__label` instead of `p`?

**Review by georgewrmarshall — COMMENTED** [review]
More of a question for design but have we explored using the expanded view real estate to view larger strings? I think ellipsis works for pop up view. If it has been explored and it doesn't work that makes sense. Just want to make sure we are designing responsively and utilizing both screen sizes effectively cc @SaraCheikh 

Screenshot 2023-07-10 at 9 19 28 AM

File: ui/components/ui/icon-button/icon-button.scss (hunk 1 above)
Comment (NidhiKJha) [reply ui/components/ui/icon-button/icon-button.scss:34]: Updated here https://github.com/MetaMask/metamask-extension/pull/19920/commits/abdb605a862c80656a4ebd45616e5e8168096973

**Comment by SaraCheikh** [comment]
@georgewrmarshall We tried to avoid displaying five buttons here to avoid this issue 🥺 We haven't used ellipsis for the labels in expanded view because there is enough space to display the label of all actions even for long translations. @NidhiKJha does this sound good to you? example of long transactions:


Screenshot 2023-07-12 at 13 43 26

**Comment by kevinghim** [comment]
For popup view, we'll use truncation and ellipsis. In Fullscreen view, we should use fix-width for the whole table, but auto-width individual buttons.

**Review by darkwing — APPROVED** [review]

**Review by danjm — APPROVED** [review]

**Pull request merged**